lib/assert.js

```diff
@@ -20,6 +24,20 @@ function test(ok, opts) {
 	}
 }
 
+function enhanceAssert(assert) {
+	return empower(assert,
+		powerAssertFormatter({
+			renderers: [
+				powerAssertRenderers.AssertionRenderer,
+				powerAssertRenderers.SuccinctRenderer
+			]
+		}),
+		{
+			destructive: true
+		}
+	);
```

Comment:
@sindresorhus Yes. It would be better to configure them explicitly.
(Is `t.assert` public API?)

```js
function enhanceAssert(assert) {
    empower(assert,
		powerAssertFormatter({
			renderers: [
				powerAssertRenderers.AssertionRenderer,
				powerAssertRenderers.SuccinctRenderer
			]
		}),
		{
			destructive: true,
			modifyMessageOnRethrow: true,
			saveContextOnRethrow: false,
			patterns: [
				't.ok(value, [message])',
				't.assert(value, [message])'
			]
		}
	);
}
```


And if you want to enhance output of assertion methods other than `ok`, you can enhance them.

```js
function enhanceAssert(assert) {
    empower(assert,
		powerAssertFormatter({
			renderers: [
				powerAssertRenderers.AssertionRenderer,
				powerAssertRenderers.SuccinctRenderer
			]
		}),
		{
			destructive: true,
			modifyMessageOnRethrow: true,
			saveContextOnRethrow: false,
			patterns: [
				't.ok(value, [message])',
				't.notOk(value, [message])',
				't.true(value, [message])',
				't.false(value, [message])',
				't.is(value, expected, [message])',
				't.not(value, expected, [message])',
				't.same(value, expected, [message])',
				't.notSame(value, expected, [message])',
				't.regexTest(regex, contents, [message])'
			]
		}
	);
}
```

In this case, intrumentation patterns in `lib/babel.js` should be the same.
```js
var options = {
	only: /(test|test\-.+|test\/.+)\.js$/,
	blacklist: hasGenerators ? ['regenerator'] : [],
	optional: hasGenerators ? ['asyncToGenerator'] : [],
	plugins: [
		createEspowerPlugin(require('babel-core'), {
			patterns: [
				't.ok(value, [message])',
				't.notOk(value, [message])',
				't.true(value, [message])',
				't.false(value, [message])',
				't.is(value, expected, [message])',
				't.not(value, expected, [message])',
				't.same(value, expected, [message])',
				't.notSame(value, expected, [message])',
				't.regexTest(regex, contents, [message])'
			]
		})
	]
};
```